src/components/TextArea/__snapshots__/TextArea.spec.js.snap

```diff
@@ -12,8 +12,6 @@ exports[`TextArea should prioritize disabled over error styles 1`] = `
   width: 100%;
   font-size: 15px;
   line-height: 24px;
-  overflow: auto;
-  resize: vertical;
```

Comment:
This doesn't seem right. The `resize: vertical;` should be applied to the textarea, not the container div.

Reply:
Perhaps this is because of the `classname` prop that you added to the input? 🤔